test/CodeGenILTests/RangeMicroTests.cs

```diff
@@ -26,6 +26,8 @@ protected virtual void GetLibrariesToPreload(ref List<string> libraries)
         [SetUp]
         public void Setup()
         {
+            //TODO_MSIL: remove the dependency on the old VM by implementing
+            //necesary Emit functions(ex mitFunctionDefinition and EmitImportStatements and all the preloading logic)
```

Comment:
Something to keep in mind is that assemblies are not just imported into the VM using import statements, there are other code paths in Dynamo currently that also preload them into the VM.

Reply:
thanks, it's a good point - @pinzart90 did you have time to file a task on addressing these TODOs? It would be good to add @aparajit-pratap's point about preloading to the task AC.

Reply:
I'll ping you @mjkkirschner  when the task is created